frontends/infinite-corridor/src/libs/react-query.test.tsx

```diff
@@ -3,9 +3,11 @@ import { renderHook, waitFor } from "@testing-library/react"
 import { allowConsoleErrors } from "ol-util/src/test-utils"
 import { QueryClientProvider } from "@tanstack/react-query"
 import { useQuery } from "@tanstack/react-query"
+import { createBrowserHistory } from "history"
 import { createQueryClient } from "./react-query"
 
-const queryClient = createQueryClient()
+const browserHistory = createBrowserHistory()
+const queryClient = createQueryClient(browserHistory)
 const wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => (
```

Comment:
This test file probably shouldn't use a global wrapper. Right now I think history and queryClient are shared between tests in this file. _My bad..._

If you want to abstract the wrapper, my rec would be:
```ts
const getWrapper = (history: MemoryHistory) => {
  const queryClient = createQueryClient(history)
  const wrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => (
  <QueryClientProvider client={queryClient}>
    <Router history={history}>
      {children}
    </Router>
  </QueryClientProvider>
}
```

Some notes:

Need to use `MemoryHistory` for tests, not BrowserHistory. Tests run in JSDOM which does not implement some browser APIs required by BrowserHistory

MemoryHistory accepts an `initialEntries` object that lets you specify initial URLs if needed.